orchestration/a2a_monitor.py

```
@@ -0,0 +1,403 @@
+A2A Monitor for WorldArchitect.AI Orchestrator
+
+Monitors A2A system health, cleans up stale registrations,
+and provides system status reporting.
+"""
+
+import json
+import time
+import logging
+import threading
+from pathlib import Path
+from typing import Dict, List, Any, Optional
```

Ruff (0.12.2)

`typing.Dict` imported but unused

Remove unused import

(F401)